client/lib/screen-title/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Internal dependencies
+ */
+import { setTitle as legacySetTitle } from './actions';
+
+/**
+ * Given a Redux store instance, subscribes to the store, updating the page
+ * when the title state changes. Currently, this dispatches through the legacy
+ * Flux actions used elsewhere in the codebase.
+ *
+ * @param {Object} store Redux store instance
+ */
+export function subscribeToStore( store ) {
```

Comment:
Both of your suggestions feel very opaque implementation wise:

- For the first one I think it ties in the _wrong way_, as we'd be implementing code in the Redux side of the application to update the old functionality. In my opinion it feels better if the Flux side updates their content after the Redux side has changed, instead of the Redux side starting to update the old Flux side.

- This feels more like hacking in than a proper solution. I know that in the current state we don't actually have a non-hack way of keeping both sides up to date, but I feel that the current method is more detached and maintainable then for example overriding the store dispatcher.